cg/meta/compress/compress.py

```diff
@@ -252,7 +270,9 @@ def update_fastq_hk(
 
     def add_fastq_hk(self, sample_id: str, fastq_first: Path, fastq_second: Path) -> None:
         """Add FASTQ files to housekeeper"""
-        fastq_tags = [sample_id, "fastq"]
```

Comment:
Is the sample_id not needed as a tag anymore? Is it redundant since we know which Housekeeper sample bundle we are in. Is there any other cg process that is looking at the sample_id tag (so we do not break anything if we remove it)?

Reply:
The processes that I find are related to "fastq" tags, and the tags prior compression is "flow_cell_name, fastq" not "sample_id, fastq". I Have now added the sample_id tag to external samples since they do not have a sequencer of ours connected to it.